spec/views/profile/index.html.slim_spec.rb

```diff
@@ -20,7 +20,8 @@
       expect(rendered).not_to have_xpath("//input[@value='Disable']")
     end
 
-    it 'contains link to delete account' do
+    xit 'contains link to delete account' do
+      pending 'temporarily disabled until we figure out the MBUN to SSN mapping'
```

Comment:
That seems to contradict the docs:  https://relishapp.com/rspec/rspec-core/v/2-0/docs/pending/pending-examples

```
describe "an example" do
  it "is implemented but waiting" do
    pending("something else getting finished")
    this_should_not_get_executed
  end
end
```